src/api/codeforces/codeforces.ts

```diff
@@ -0,0 +1,18 @@
+import axios from 'axios'
+import { UpcomingContestResponse, ContestResponseSchema } from './interfaces'
+
+export async function upcomingContests (): Promise<UpcomingContestResponse> {
+  const response = await axios.get('https://codeforces.com/api/contest.list')
+  const res = response.data
+  const cf: Array<ContestResponseSchema> = []
+  for (const item of res.result) {
+    if (item.phase === 'BEFORE') {
+      cf.push(item)
+    } else {
+      console.log('Oops, looks like there are no upcoming contests currently.')
+    }
```

Comment:
Handle this and send appropriate error message. In this case, user will still get empty array in the bot